manual/installation.tex

```diff
@@ -561,6 +561,46 @@ \subsection{Installing on Mac OS X using Homebrew (brew)}
 \end{verbatim}
```

Comment:
Could you put this manual section (and associated changes) in a separate pull request? It's not related to getting ppconvert into the build system